sample/sample35-one-to-one-cascade-deletion/entity/User.ts

```diff
@@ -0,0 +1,16 @@
+import { Entity, PrimaryGeneratedColumn, Column, OneToOne, JoinColumn } from "../../../src/index";
+import { Profile } from "./Profile";
+
+@Entity()
+export class User {
+
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  name: string;
+
+  @OneToOne(() => Profile, (profile: { user: any; }) => profile.user, { eager: true, onDelete: "CASCADE" })
```

Comment:
why do you need `(profile: { user: any; })` ? why not just `profile => profile.user` ?

Reply:
Thanks, I'll fix it.